relay-metrics/src/protocol.rs

```diff
@@ -67,8 +234,11 @@ impl Default for MetricUnit {
 impl fmt::Display for MetricUnit {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            MetricUnit::Duration(precision) => precision.fmt(f),
-            MetricUnit::None => f.write_str(""),
+            MetricUnit::Duration(u) => u.fmt(f),
+            MetricUnit::Information(u) => u.fmt(f),
+            MetricUnit::Fraction(u) => u.fmt(f),
+            MetricUnit::Custom(u) => u.fmt(f),
+            MetricUnit::None => f.write_str(""), // TODO: Change?
```

Comment:
We've serialized the empty unit as empty string `""` in the past. Do we want to keep this for brevity or write an explicity `none`?